app/konnect/manage-plugins/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Manage Plugins
+no_version: true
+---
+
+Any {{site.ee_gateway_names}} plugins supported in a self-managed hybrid mode
+deployment are also accessible through ServiceHub or Shared Config.
+
+## Kong plugins in Konnect Cloud
+
+### Plugin configuration
+
+You can configure a plugin in {{site.konnect_saas}} by scoping it to an object,
+or applying it globally.
+
+* A **scoped** plugin applies configuration only to a specific service, route,
+or consumer. You can configure plugins on
+[services](/konnect/manage-plugins/enable-service-plugin) and
+[routes](/konnect/manage-plugins/enable-route-plugin) through ServiceHub, and on
+[consumers](/konnect/manage-plugins/shared-config) through Shared Config.
+
+* If you want to apply a plugin **globally** &ndash; that is, to all services,
+routes, and consumers in a cluster &ndash; set it up through
+[shared config](/konnect/manage-plugins/shared-config/).
+
+### Functionality differences from self-managed Kong Gateway
+
+The Portal Application Registration plugin is enabled automatically when you
+[enable application registration](/konnect/dev-portal/administrators/app-registration/enable-app-reg)
+for the Dev Portal. You don't need to enable it directly.
+
+Conversely, don't delete the Application Registration plugin directly.
+[Disable application registration](/konnect/dev-portal/administrators/app-registration/disable-app-reg)
+from the service overview only.
+
```

Comment:
This is not actually correct. We don't even use the Application Registration plugin in Konnect Cloud. We simply enabled the ACL and auth plugins (either key auth or OIDC) behind the scenes and then don't allow customer to modify the settings of those two plugins.